test/unit/asrouter/ASRouterTargeting.test.js

```diff
@@ -98,3 +98,26 @@ describe("ASRouterTargeting#isBelowFrequencyCap", () => {
     });
   });
 });
+
+describe("ASRouterTargeting#isInExperimentCohort", () => {
```

Comment:
we also don't really need this test, since i wrote a mochitest too that does basically this... but we can never have too many tests!!!